node/nodeutil/auth.go

```diff
@@ -50,7 +51,7 @@ func InstrumentAuth(auth Auth) Auth {
 // NoAuth creates an Auth which allows anonymous access to all resouorces
 func NoAuth() Auth {
 	return &authWrapper{
-		Request:                 anonymous.NewAuthenticator(),
+		Request:                 anonymous.NewAuthenticator([]apiserver.AnonymousAuthCondition{}),
```

Comment:
Interesting. [According to the docs](https://pkg.go.dev/k8s.io/apiserver/pkg/authentication/request/anonymous#NewAuthenticator):
>When conditions is empty all requests are authenticated as anonymous.